src/libaudqt/audqt.cc

```diff
@@ -50,15 +106,7 @@ EXPORT void init ()
     static int dummy_argc = 1;
     static char * dummy_argv[] = {app_name, nullptr};
 
-    auto qapp = new QApplication (dummy_argc, dummy_argv);
-
-    qapp->setAttribute (Qt::AA_UseHighDpiPixmaps);
-#if QT_VERSION >= QT_VERSION_CHECK(5, 3, 0)
-    qapp->setAttribute (Qt::AA_ForceRasterWidgets);
-#endif
-
-    qapp->setApplicationName (_("Audacious"));
-    qapp->setWindowIcon (audqt::get_icon (app_name));
+    qapp = new AudApplication (dummy_argc, dummy_argv);
```

Comment:
The AudApplication subclass seems unnecessary; please revert it so we can see the functional changes more easily.